pkg/admission/server_test.go

```diff
@@ -285,9 +285,9 @@ func TestServeHTTPSubmissions(t *testing.T) {
    								               }
    								            }
    								         ],
-   								         "forwardTo": [
```

Comment:
tbh kinda worrying that this was passing the tests. I wonder if we're not running this one?